db/seeds.rb

```diff
@@ -2,7 +2,7 @@
   password: '123456',
   full_name: 'Admin test',
   cpf: FFaker::IdentificationBR.cpf,
-  cellphone: FFaker::PhoneNumber.short_phone_number,
+  cellphone: rand.to_s[2..16],
```

Comment:
I think we can use some method from `faker` gem, maybe something like: `FFaker.numerify('###############')`, it looks easier to understand :D

Reply:
Ok, got it